docs/UI/PerformInteraction/index.md

```diff
@@ -39,6 +39,7 @@ A UI.PerformInteraction with a timeout value of `0` should not be timed out imme
 |timeout|Integer|true|minvalue: 5000<br>maxvalue: 100000|
 |interactionLayout|[Common.LayoutMode](../../common/enums/#layoutmode)|false||
 |appID|Integer|true||
+|cancelID|Integer|false||
```

Comment:
Might need to include this in VR.CancelInteraction as well.

Reply:
added in https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/163/commits/1fa4a7c385df4f2eb38fb64a088c24f5a73ee05b